pandas/tests/groupby/test_grouping.py

```diff
@@ -564,52 +563,6 @@ def test_list_grouper_with_nat(self):
 
 class TestGetGroup():
 
-    @pytest.mark.filterwarnings("ignore:\\nPanel:FutureWarning")
-    def test_get_group(self):
-        wp = tm.makePanel()
-        grouped = wp.groupby(lambda x: x.month, axis='major')
-
-        gp = grouped.get_group(1)
-        expected = wp.reindex(
-            major=[x for x in wp.major_axis if x.month == 1])
-        assert_panel_equal(gp, expected)
-
-        # GH 5267
-        # be datelike friendly
-        df = DataFrame({'DATE': pd.to_datetime(
```

Comment:
Hmm should we keep this DataFrame test around?

Reply:
Good catch.  It isn't always obvious when a multi-part test has pieces that can be salvaged.  Will update.